### PR TITLE
feat: Make.com integration

### LINKED ISSUE #599: [FEATURE] Build make.com integration
**Is your feature request related to a problem? Please describe.**
As a user I want to pipe my insights directly to Google Sheets

**Describe the solution you'd like**
Build a direct Make.com integration

**Tasks**
1. Look at how other tools do their Make.com integrations and see what you like from an UX point of view
2. Take the best from everything and ship it :)
3. Add it to the Integrations page on Formbricks

Thanks for picking this up! 🚀

**Additional context**

### How we code at Formbricks 🤓

- Everything is type-safe
- All UI components are in the package `formbricks/ui`
- Run `pnpm dev` to find a demo app to test in-app surveys at `localhost:3002`
- We use **chatGPT** to help refactor code. Use our [Formbricks ✨ megaprompt ✨](https://github.com/formbricks/formbricks/blob/main/megaprompt.md) to create the right context before you write your prompt.

### PR DESCRIPTION
**Pull request by Dhruwang**
## What does this PR do?

Fixes #599 

1. Adds make to formbricks Integration page
2. Adds a new section for make.com in documentation
3. Created an app for formbricks on make.com (under review) - [Apps public link](https://eu2.make.com/app/invite/67538c04a8ed934e86d4ba8cdd8f8732)

Screenshot from 2023-08-01 12-03-49

make

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [x] This change requires a documentation update

## How should this be tested?

Check Integration page and Integration docs
[Check app on make.com](https://eu2.make.com/app/invite/67538c04a8ed934e86d4ba8cdd8f8732)

## Checklist

- [x] Added a screen recording or screenshots to this PR
- [x] Filled out the "How to test" section in this PR
- [x] Read the [contributing guide](https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues
- [x] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/(app)/environments/[environmentId]/integrations/page.tsx [hunk 1]
```diff
@@ -22,6 +23,13 @@ export default function IntegrationsPage() {
           description="Integrate Formbricks with 5000+ apps via Zapier"
           icon={<Image src={ZapierLogo} alt="Zapier Logo" />}
         />
+        <Card
+          docsHref="https://formbricks.com/docs/integrations/make"
+          connectHref="https://zapier.com/apps/formbricks/integrations"
```
Comment (Dhruwang) [inline apps/web/app/(app)/environments/[environmentId]/integrations/page.tsx:28]: Will replace the link with the make one, once app is approved

**Comment by mattinannt** [comment]
@jobenjada What's the status on this?

**Comment by Dhruwang** [comment]
@mattinannt Its ready for review :)

**Review by mattinannt — APPROVED** [review]
@Dhruwang Great job 🤗💪

**Pull request merged**